cpp/include/nvstrings/NVText.h

```diff
@@ -55,6 +55,17 @@ class NVText
      */
     static NVStrings* unique_tokens(NVStrings& strs, const char* delimiter=nullptr);
 
+    /**
+     * @brief Tokenize all the strings into a single characters only.
+     * This will create a single character per string entry in the returned instance.
```

Comment:
Hmm, does it produce a single character per input string, or a single string per input character? Rephrase for clarity.

```suggestion
    *
    * This creates a single character per string entry in the returned instance.
```